src/network/connection.spec.js

```diff
@@ -98,6 +98,62 @@ describe('Network > Connection', () => {
       expect(connection.socket.end).toHaveBeenCalled()
       expect(connection.socket.unref).toHaveBeenCalled()
     })
+
```

Comment:
@fbonecco-cribl This gives me an error about "unnecessary escape" when I try to run the tests.

Reply:
Thanks, should be fixed now.